pkg/process/events/model_linux.go

```diff
@@ -0,0 +1,43 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build linux
+// +build linux
+
+package events
+
+import "time"
+
+// ProcessEvent represents a process event collected by system-probe
+type ProcessEvent struct {
```

Comment:
As JSON serialization is pretty costly, using like messagepack may be useful (it is already used by remote configuration so it's pretty much just creating a .proto file for the process events)

Reply:
That's a really good point! I've updated the PR to use messagepack instead of JSON and added benchmarks for it. Thanks for the suggestion!
```
$ go test -bench=.
goos: linux
goarch: amd64
pkg: github.com/DataDog/datadog-agent/pkg/process/events/model
cpu: Intel(R) Core(TM) i7-8569U CPU @ 2.80GHz
BenchmarkProcessEventsJSON10-4        	   36254	     28897 ns/op
BenchmarkProcessEventsMsgPack10-4     	  371541	      2954 ns/op
BenchmarkProcessEventsJSON100-4       	   23048	     51829 ns/op
BenchmarkProcessEventsMsgPack100-4    	  182128	      6862 ns/op
BenchmarkProcessEventsJSON1000-4      	    4712	    242861 ns/op
BenchmarkProcessEventsMsgPack1000-4   	   28173	     42777 ns/op
**PASS**
```

Reply:
Thanks for the benchmarks !

Reply:
You're welcome!